source/application/SGM/SGMAppController.cpp

```diff
@@ -30,6 +30,9 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 #include "ui/CLS/CLSSIS3820ScalerView.h"
 #include "acquaman/AMGenericStepScanConfiguration.h"
 #include "ui/acquaman/AMGenericStepScanConfigurationView.h"
+#include "util/AMErrorMonitor.h"
+
+#include <stdlib.h> // Used for obtaining username to prevent users other than iain (for dev) or SGM-Upgrade (for commissioning). Remove for deploy.
```

Comment:
Yeah, it's just there for the getenv("USER") function, which acts as a check to stop someone, somehow managing to run the development version on an account which has old acquaman data.